scripts/phylum-uninstall.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+# Don't continue after failure:
+set -eu
+
+success() {
+    printf "%b    OK%b %s\n" "${GREEN}" "${NC}" "${1}"
+}
+
+banner() {
+    printf "\n    %bphylum-cli%b uninstaller\n\n" "${GREEN}" "${NC}"
+}
+
+get_rc_file() {
+    case $(basename "${SHELL}") in
+        bash)
+            echo "${HOME}/.bashrc"
+            ;;
+        zsh)
+            echo "${HOME}/.zshrc"
+            ;;
+        fish)
+            echo "${HOME}/.config/fish/config.fish"
+            ;;
+        *)
+            echo "shell's configuration file"
+            ;;
+    esac
+}
+
+remove_from_path_and_alias() {
+    rc_path=${1}
+    shell=${2}
+
+    if grep -q ".phylum:\$PATH" "${rc_path}"; then
+        # shellcheck disable=SC2016 # Expanding variables here would be a bad idea
+        sed -i '/^export PATH="$HOME\/.phylum:$PATH"$/d' "${rc_path}"
+        success "Removed ${HOME}/.phylum/ from ${shell}'s \$PATH."
+    fi
+
+    if grep -q 'alias ph=' "${rc_path}"; then
+        sed -i "/^alias ph='phylum'$/d" "${rc_path}"
+        success "Removed ph alias for phylum from ${shell}."
+    fi
+}
+
+cleanup_zshrc() {
+    rc_path="${HOME}/.zshrc"
+
+    if grep -q '.phylum/completions' "${rc_path}"; then
+        # shellcheck disable=SC2016 # Expanding variables here would be a bad idea
+        sed -i '/^fpath+=("$HOME\/.phylum\/completions")$/d' "${rc_path}"
+    fi
+    if grep -q 'autoload -U compinit && compinit' "${rc_path}"; then
+        sed -i '/^autoload -U compinit && compinit$/d' "${rc_path}"
+    fi
+    success "Removed completions from zsh config."
+
+    remove_from_path_and_alias "${rc_path}" "zsh"
+}
+
+cleanup_bashrc() {
+    rc_path="${HOME}/.bashrc"
+
+    if grep -q '.phylum/completions/phylum.bash' "${rc_path}"; then
+        # shellcheck disable=SC2016 # Expanding variables here would be a bad idea
+        sed -i '/^source $HOME\/.phylum\/completions\/phylum.bash$/d' "${rc_path}"
```

Comment:
There is an error on this line...but not familiar enough with `sed` to say what:
```sh
❯ uninstall-phylum.sh --purge

    phylum-cli uninstaller

    OK Removed all files.
sed: 1: "/Users/coggins/.bashrc": command c expects \ followed by text
```
BTW...the name `uninstall-phylum.sh` here is just b/c I renamed the file in the `install.sh` script while trying all this out.